.github/scripts/ffmpeg/execute-tests.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+source `dirname ${BASH_SOURCE[0]}`/../config.sh
+
+SOURCE_PATH=$(realpath `dirname ${BASH_SOURCE[0]}`/../../../code)
+SKIP_TESTS_PATH=$(realpath `dirname ${BASH_SOURCE[0]}`/skip_tests.txt)
+
+echo "::group::Test FFmpeg"
+    pushd $FFMPEG_TESTS_PATH
+
+    mkdir -p tests/data/fate
+    ln -s /usr/bin/base64 tests/base64
+
+    run_test=0
+    IFS=$'\n'
+    for i in $(sed '/^@/d;/fate-source/d' fate.log); do
+        if [[ $i =~ ^TEST ]]; then
+            test_name=${i##* }
+            run_test=$(cat $SKIP_TESTS_PATH | grep -q "^$test_name$" && echo 0 || echo 1)
+            [[ $run_test == 1 ]] && echo TEST $test_name || echo SKIP $test_name
+            continue
+        fi
+
+        if [[ $run_test == 0 ]] || [[ ! $i =~ ^echo ]]; then
+            continue
+        fi
+
+        [[ $run_test == 1 ]] && echo $i
```

Comment:
I'd remove this line. I guess it was good for debugging but effectively does nothing and brings just noise to the output especially when the paths of the builds machine are different that paths on the tests execution machine.

Reply:
yes, refactoring was need here